testsuite/features/support/commonlib.rb

```diff
@@ -194,7 +194,9 @@ def suse_host?(name)
 end
 
 def slemicro_host?(name)
-  (name.include? 'slemicro') || (name.include? 'micro')
+  node = get_target(name)
+  _os_version, os_family = get_os_version(node)
+  (name.include? 'slemicro') || (name.include? 'micro') || os_family.include?('sle-micro')
```

Comment:
```
os_family.include?('sle-micro')
```
only? or even remove the function and check `os_family` in the caller code?

Reply:
```ruby
os_family.include?('sle-micro')
```
will only partially wok because this is only valid for SLE Micro >= 5.3. If we want to have it for all supported versions, we need to change it e.g. to

```ruby
os_family.include?('sle-micro') || os_family.include?('suse-microos')
```

Reply:
I do not want to remove the first two checks, since I did not test them. But I added the 2 others I mentioned above.

Reply:
ok